src/plots/cartesian/axes.js

```diff
@@ -2064,6 +2069,10 @@ function beyondSI(exponent) {
     return exponent > 14 || exponent < -15;
 }
 
+function beyondSIExtended(exponent) {
+    return exponent > 32 || exponent < -32;
```

Comment:
`> 32` I think is right, as we’d display up to `100Q`, but at the bottom shouldn’t it be `< -30`, as `1q` will be the smallest?

Reply:
You're completely right, my bad! For exponents smaller than `-30` we need to display the full power.